### DISCUSSION TITLE
what hourly forecast data can we get

### DISCUSSION THREAD

**Discussion by ghost** [weather-gov/api#230] [Q&A]
From the examples we get:

    periods": [
            {
                "number": 1,
                "name": "",
                "startTime": "2021-03-01T15:00:00-06:00",
                "endTime": "2021-03-01T16:00:00-06:00",
                "isDaytime": true,
                "temperature": 51,
                "temperatureUnit": "F",
                "temperatureTrend": null,
                "windSpeed": "5 mph",
                "windDirection": "N",
                "icon": "https://api.weather.gov/icons/land/day/skc?size=small",
                "shortForecast": "Sunny",
                "detailedForecast": ""
            },

are these the only items we can get forecasts for?
Also, for short forecast and wind direction is there a complete list of responses?

was wondering if there is wave information, probability of rain, probability of thunder, barometer and dew point?

**Comment by JJKraw** [comment]
Have a look at the /gridpoints/{wfo}/{x},{y} endpoint. It's raw data, not collected together as nicely as the hourly endpoint, but there is a lot more information there.

**Comment by scadergit** [comment]
Enhancing the API forecast endpoints is overdue, sorry. The application was just decoupled from the deployment schedule of the overall platform to hopefully support more frequent updates.

**Reply by lymanepp** [reply]
@scadergit, has any progress been made on adding more attributes to the hourly forecast?
